Server/controllers/time-entry.controller.js

```diff
@@ -4,13 +4,124 @@
  *
  * @class timeEntriesController
  */
+const mongoose = require('mongoose');
+const timeEntryModel = require('../models/time-entry.model');
```

Comment:
I think we don't need model in controller

Reply:
Updated the code by instantiating Model in Datalayer